src/Uno.UI/ts/Windows/System/Profile/AnalyticsInfo.ts

```diff
@@ -0,0 +1,59 @@
+﻿namespace Windows.System.Profile {
+
+	export class AnalyticsInfo {
+		public static getDeviceType(): string {
```

Comment:
Codacy found an issue: [The function getDeviceType has a cyclomatic complexity of 39 which is higher than the threshold of 20](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6396919)